### PR TITLE
Pricing page update

### PR DESCRIPTION
**Pull request by Woafflation**
Added Free plan
UI fixe
Снимок экрана 2022-06-27 в 12 47 11
Снимок экрана 2022-06-27 в 12 47 52
Снимок экрана 2022-06-27 в 12 48 21
Снимок экрана 2022-06-27 в 12 49 19
s

### CODE REVIEW

File: src/ui/Pricing/Comparison/Feature.svelte [hunk 1]
```diff
@@ -26,7 +26,7 @@
       <div slot="trigger" class="info btn mrg-xs mrg--l row hv-center">
         <Svg id="info" w="12" />
       </div>
-      <div slot="tooltip" class="description">{@html description}</div>
+      <div slot="tooltip" class="description body-3">{@html description}</div>
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/Comparison/Feature.svelte:29]: `body-3` is inherited from the body and applied by default

File: src/ui/Pricing/Comparison/index.svelte [hunk 2]
```diff
@@ -42,6 +48,7 @@
       {/if}
     </div>
   </Table>
+  <FeaturesToggle bind:active={isShowLess} onClick={showMoreFeatures} />
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/Comparison/index.svelte:51]: 1. Name is misleading and vaguely represent what this component does. Maybe, we can name it something like `<MoreFeaturesButton />`?
2. Use `bind` directive -> `bind:isShowingMore`

File: src/ui/Pricing/Comparison/index.svelte [hunk 3]
```diff
@@ -2,12 +2,14 @@
...
   import Slides from '@/ui/Slides.svelte'
   import Table from './Table.svelte'
+  import FeaturesToggle from './FeaturesToggle.svelte'
   import Plan from './Plan.svelte'
   import { PlanFeatures } from './comapre'
 
   let className = ''
   export { className as class }
   export let plans: SAN.Plan[]
+  export let isShowLess = true
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/Comparison/index.svelte:12]:
```suggestion
  export let isShowingMore = false
```

File: src/ui/Pricing/Comparison/FeaturesToggle.svelte [hunk 4]
```diff
@@ -0,0 +1,32 @@
+<script>
+  import Svg from '@/ui/Svg/svelte'
+
+  export let onClick = () => {}
+  export let active = false
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/Comparison/FeaturesToggle.svelte:5]:
```suggestion
  export let isShowingMore = false
```

File: src/ui/Pricing/Comparison/FeaturesToggle.svelte [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+</script>
+
+<div class="wrapper fluid row hv-center" on:click={onClick}>
+  <div class="fluid btn-3 more body-2 mrg--b mrg--t mrg-m">
+    Show {active ? 'more' : 'less'} features
+    <div class:showMore={!active} class="row v-center mrg-m mrg--l">
+      <Svg id="arrow-down" w="10" h="5.5" />
+    </div>
+  </div>
+</div>
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/Comparison/FeaturesToggle.svelte:15]:
```suggestion
<button class="btn btn--accent row hv-center" class:more={isShowingMore} on:click={() => (isShowingMore = !isShowingMore)}>
    Show {isShowingMore ? 'less' : 'more'} features
    <Svg id="arrow-down" w="10" h="5.5" class="$style.arrow mrg-m rgm--l" />
</button>
```

File: src/ui/Pricing/Comparison/FeaturesToggle.svelte [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    }
+
+    .showMore {
+      transform: rotate(180deg);
+    }
+
+    &:hover {
+      background: var(--porcelain);
+    }
+  }
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/Comparison/FeaturesToggle.svelte:31]:
```suggestion
 button {
   padding: 12px;
   --bg-hover: var(--porcelain);
 }
 
 .more {
   --rotate: 180deg;
 }
 
 .arrow {
   transform: rotate(var(--rotate));
 }
```

File: src/ui/Pricing/Comparison/Plan.svelte [hunk 7]
```diff
@@ -9,26 +9,30 @@
   export let isEligibleForTrial: boolean
   export let isLoggedIn = false
 
-  $: ({ name } = plan)
+  $: ({ name: planName } = plan)
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/Comparison/Plan.svelte:12]:
```suggestion
  $: ({ name } = plan)
```

File: src/ui/Pricing/Comparison/comapre.ts [hunk 8]
```diff
@@ -176,12 +210,15 @@ export const PRO_COMPARED_FEATURES = {
 }
 
 export const PRO_PLUS_COMPARED_FEATURES = {
+  'Bi-weekly report': true,
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/Comparison/comapre.ts:213]: This is a default value
```suggestion
```

File: src/ui/Pricing/Comparison/comapre.ts [hunk 9]
```diff
@@ -159,8 +163,38 @@ export const COMPARE_TABLE = [
...
+  Screeners: 1,
+  'Screener filters': 'Limited',
+
+  'Google Sheets Plugin': false,
+  CSV: false,
+}
+
+export const PRO_COMPARED_FEATURES = {
+  'Browser tabs': 'Unlimited',
+  'Bi-weekly report': true,
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/Comparison/comapre.ts:197]: These are default values

```suggestion
```

**Review by DmitriVanGuard — CHANGES_REQUESTED** [review]

File: src/ui/Pricing/Comparison/Feature.svelte (hunk 1 above)
Comment (Woafflation) [reply src/ui/Pricing/Comparison/Feature.svelte:29]: Снимок экрана 2022-06-28 в 15 52 13
Снимок экрана 2022-06-28 в 15 52 20

Have to manually apply it

File: src/styles/buttons.css [hunk 10]
```diff
@@ -117,6 +117,11 @@ button,
   --border-hover: var(--orange-hover);
 }
 
+.btn--accent {
+  --color: var(--green);
+  --color-hover: var(--green-hover);
+}
```
Comment (DmitriVanGuard) [inline src/styles/buttons.css:123]: Accent color should depend on the `--accent` variable as it does in the [`.c-accent` class](https://github.com/santiment/san-webkit/blob/master/src/styles/text.css#L94-L96)

File: src/ui/Pricing/Comparison/Table.svelte [hunk 11]
```diff
@@ -1,15 +1,38 @@
...
+
+  function filterTableCompareItems(items, shouldFilter) {
+    if (shouldFilter) {
+      return COMPARE_TABLE.slice(0, 7)
+    }
+
+    return COMPARE_TABLE
+  }
+
+  $: tableItems = filterTableCompareItems(plans, isShowingMore)
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/Comparison/Table.svelte:17]: Reactive statements should come before the function declarations
```suggestion
  $: tableItems = filterTableCompareItems(plans, isShowingMore)
  
  function filterTableCompareItems(items, shouldFilter) {
    if (shouldFilter) {
      return COMPARE_TABLE.slice(0, 7)
    }

    return COMPARE_TABLE
  }
```

File: src/ui/Pricing/PlanButton.svelte [hunk 12]
```diff
@@ -92,4 +96,10 @@
     --v-padding: 7px;
     --color-hover: var(--accent);
   }
+
+  .free {
+    --bg: var(--green);
+    --color: var(--white);
+    --bg-hover: var(--green-hover);
+  }
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/PlanButton.svelte:104]: Duplicate of the `btn--green` class

**Review by DmitriVanGuard — CHANGES_REQUESTED** [review]

**Review by DmitriVanGuard — CHANGES_REQUESTED** [review]

**Pull request merged**